### DISCUSSION TITLE
Execute Pimcore offline

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#17024] [Q&A / Support]
As soon as Pimcore has been installed, I can logically run it offline (without an internet connection).
What functional limitations should I actually expect then?

**Comment by ghost** [comment]
The question is serious, btw. I tried something around. I can work normally in the admin area.

**Comment by markus-moser** [comment]
Do you mean hosted by a local server (e.g. docker based)? This should work without a problem and should not have really relevant limitations.
